lib/confirmation.rb

```diff
@@ -0,0 +1,20 @@
+class Confirmation
+
+	attr_reader :checkout, :total
+
+	def initialize(checkout)
+		@checkout = checkout
+		@total = 0
+	end
+	
+	def user_cost (user_total)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)